src/Integration.UnitTests/Service/SonarQubeServiceWrapperTests.cs

```diff
@@ -464,6 +464,39 @@ public void SonarQubeServiceWrapper_GetPlugins()
             }
         }
 
+        [TestMethod]
+        public void SonarQubeServiceWrapper_CreateProjectDashboardUrl()
+        {
+            // Setup
+            var testSubject = new SonarQubeServiceWrapper(this.serviceProvider);
+
+            var serverUrl = new Uri("http://my-sonar-server:5555");
+            var connectionInfo = new ConnectionInformation(serverUrl);
+            var projectInfo = new ProjectInformation { Key = "p1" };
+
+            Uri expectedUrl = new Uri(serverUrl, string.Format(SonarQubeServiceWrapper.ProjectDashboardRelativeUrl, projectInfo.Key));
```

Comment:
It's perhaps better just to supply a hard-coded string for the URL that's expected: it's easier to read, and it actually checks that the correct string is returned.
As it is, the test will check that the string format used in the service wrapper is used, but that doesn't help if the string format is wrong.
Using a hard-coded constant in the test does of course mean that the test might have to change if the expected url changes, but I don't see that as much of an issue. Fixing up the test would be trivial, and it would provide the prompt to think about any other tests that might be required.